core/shared/src/main/scala/chartreuse/TickMarkCalculator.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015 Creative Scala
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package chartreuse
+
+import scala.math.{log10, ceil, floor, pow}
+
+object TickMarkCalculator {
+  def calculateTickScale(
+      min: Double,
+      max: Double,
+      tickCount: Double
+  ): (Double, Double, Double) = {
```

Comment:
A tuple isn't a good choice to return here, as from the users point of view there are three `Double` values and there is no information to distinguish between these values. Which one is the min, which one the max, etc? The only way to know is to read the code.

Better to create a simple type

```scala
final case class Ticks(min: Double, max: Double, size: Double)
```